src/DynamoCore/Core/Threading/Disposable.cs

```diff
@@ -3,7 +3,7 @@
 using System.Linq;
 using System.Text;
 
-namespace Dynamo.Utilities
+namespace Dynamo.Core.Threading
```

Comment:
@lukechurch, assuming you meant "Why is this in threading..." Because that's the only place that it is used.  It's in `AsyncTaskExtensions`.